site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Topology Aware Scheduling"
+date: 2024-04-11
+weight: 6
+description: >
+  Allows scheduling of Pods based on the topology of nodes in a data center.
+---
+
+{{< feature-state state="alpha" for_version="v0.9" >}}
+
+It is common that AI/ML workloads require a significant amount of pod-to-pod
+communication. Therefore the network bandwidth between the running Pods
+translates into the workload execution time, and the cost of running
+such workloads. The available bandwidth between the Pods depends on the placement
+of the Nodes, running the Pods, in the data center.
+
+We observe that the data centers have a hierarchical structure of their
+organizational units, like racks and blocks, where there are multiple nodes
+within a rack, and there are multiple racks within a block. Pods running within
+the same organizational unit have better network bandwidth than Pods on
+different units. We say that nods placed in different racks are more distant
+than nodes placed within the same rack. Similarly, nodes placed in different
+blocks are more distant than two nodes within the same block.
+
+In this feature (called Topology Aware Scheduling, or TAS for short) we
+introduce a convention to represent the
+[hierarchical node topology information](#node-topology-information), and a set
+of APIs for Kueue administrators and users to utilize the information
+to optimize the Pod placement.
+
+### Node topology information
```

Comment:
Could we mention that we only care about ready nodes?

Reply:
done